docs/effects.md

```diff
@@ -30,3 +46,6 @@ square.addEffect(ScaleEffect(
 ));
 ```
 
+# Examples
+
+Full examples can be found [here](./examples/effects/simple) and [here](./examples/effects/infinite_effects).
```

Comment:
This path is not correct right?

Reply:
I think it is?

Reply:
shouldn't it be `/docs/examples/effects/infinite_effects`? So the site generate script can work properly?

Reply:
Of course, forgot that, will fix! My IDE put the path in there automagically.